CMakeLists.txt

```diff
@@ -129,7 +129,7 @@ ELSE()
 ENDIF()
 
 SET_TARGET_PROPERTIES(pthreadpool PROPERTIES
-  C_STANDARD 11
+  C_STANDARD 14
```

Comment:
his line introduces an error for me. This is the output from cmake 3.29.2 for 472fc764ee0ae9be294cf054e21b04555c60db35 applied to https://github.com/Maratyszcza/pthreadpool/commit/178e3e0646cc671708bf78e77c273940130ac637:
```
CMake Error at CMakeLists.txt:103 (ADD_LIBRARY):
  C_STANDARD is set to invalid value '14'


-- Generating done (0.0s)
```
Dcoumented values  for [`C_STANDARD`](https://cmake.org/cmake/help/v3.29/prop_tgt/C_STANDARD.html) for 3.29 do not include 14. Removing this line allows building with 1.14.0.